modules/core/test/v2/integration/bitgo.ts

```diff
@@ -71,13 +71,6 @@ describe('BitGo', function() {
     });
 
     describe('Logout API', function() {
-      it('arguments', function(done) {
```

Comment:
why delete?

Reply:
It checks to make sure that it throws if a non-function is passed as the callback arg, which causes other problems regarding dual error channels (long story). I'll address that in the PR description as this is a known breaking change (next version will be 8.0.0).